examples/CMakeLists.txt

```diff
@@ -9,28 +9,6 @@
 
 cmake_minimum_required(VERSION 3.1)
 
-## serialbox_add_c_example
-## -----------------------
-##
-## Create an install a C example
-##
-## NAME:STRING=<>     - Name of target and executable
-## ARGN:STRINGS=<>    - List of source files
-##
-function(serialbox_add_c_example NAME)

```

Comment:
yoda provides a complete `yoda_add_unittest` that allows to specify properly dependencies and includes. There is no need for all these `serialbox_add_c_example` and others here... But I only cleaned up this one

See here how to use it: 
https://github.com/eth-cscs/serialbox2/pull/167/files#diff-961777772df9b51c4517f3cfe0580c97R12